### PR TITLE
Feature/update mixpanel version ios to 4.1.5

### PR DESCRIPTION
**Pull request by vrenan**
Upgrade the Mixpanel iOS version to 4.1.5.

### CODE REVIEW

File: src/ios/updateVersion.sh [hunk 1]
```diff
@@ -18,7 +18,8 @@ curl -Ls https://github.com/mixpanel/mixpanel-iphone/archive/$ZIPFILE -O $ZIPFIL
 unzip $ZIPFILE
 rm -f $ZIPFILE
 mv Mixpanel Mixpanel_bak
-mv mixpanel-iphone-$VERSION/Mixpanel .
+mv mixpanel-iphone-$VERSION/Sources .
+mv Sources Mixpanel
```
Comment (samzilverberg) [inline src/ios/updateVersion.sh:22]: suggestion: combine into 1 line :)
```suggestion
mv mixpanel-iphone-$VERSION/Sources ./Mixpanel
```

File: src/ios/MixpanelPlugin.h [hunk 2]
```diff
@@ -34,7 +34,6 @@
 -(void)people_append:(CDVInvokedUrlCommand*)command;
 -(void)people_deleteUser:(CDVInvokedUrlCommand*)command;
 -(void)people_increment:(CDVInvokedUrlCommand*)command;
--(void)people_setPushId:(CDVInvokedUrlCommand*)command;
```
Comment (samzilverberg) [inline src/ios/MixpanelPlugin.h:37]: this removal (by mixpanel lib) might be a problem.
the cordova plugin currently exposes `window.mixpanel.people.setPushId` and expects it to be available for both android & ios :( .

we'll need to adapt `setPushId` a little so existing code that uses this on ios doesn't break suddenly.

dya know of any documentation saying why was this removed? maybe was replaced with something else?

**Comment by samzilverberg** [comment]
super thanks for the contribution.
I hardly find time to do anything here nowadays.
and also figured cordova is so old by now that everyone has abandoned it 😆 

I'm a little worried that `setPushId` was removed for ios, but is still used by the plugin code.
we can probably find a workaround for this.

**Comment by jpike88** [comment]
@samzilverberg please mark your repo as not maintained, or looking for maintainers. Cordova is alive and well, there are plenty of us out there using it. This plugin is getting downloaded >500 times a day on npm, I'd say that's a good indicator there's healthy usage.

**Comment by jpike88** [comment]
fyi I have opened a separate PR updating android, it removes the need to pull in firebase anymore, as well as removes pushID as well. as @vrenan has not actioned your suggestions I will open another PR to get this speeding along.

**Comment by jpike88** [comment]
> super thanks for the contribution. I hardly find time to do anything here nowadays. and also figured cordova is so old by now that everyone has abandoned it 😆
> 
> I'm a little worried that `setPushId` was removed for ios, but is still used by the plugin code. we can probably find a workaround for this.

I have simply caused the function to call onFail instead, with relevant link explaining why.

It's probably worth updating this repo to the next major version, and putting a warning up the top of the readme.

**Comment by samzilverberg** [comment]
@jpike88 sorry for the long delay. got very sick in the last 2 weeks 😢 
I finally had some time on the weekend to try and get back on this project but failed miserably.
Installing the android sdk and other deps (on mac) just took too long and required too much troubleshooting that I ran out of time to continue.

so wasn't able to `cordova build` or test anything.

i'll just merge this PR as is, and will also mark the repo as unmaintained/looking for maintainer as you advised, because I don't see myself having time anytime soon to fully get the dev env working and maintain this again :( very sad.....
